week3/homework/src/route/todos.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+const Todos = require('../main');
+
+const todos = new Todos('./todos.json');
+
+router.get('/', (req, res) => todos.realTodos(req, res));
```

Comment:
I think you are missing the `resource` part of the REST URL. What is the client trying to get?